Frontend/src/components/AdminSettings/AdminSettings.jsx

```diff
@@ -1,7 +1,88 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import RequestService from '../../util/RequestService';
+import { CirclePicker } from 'react-color';
+import './AdminSettings.css';
+
+const deleteColorFromDatabase = color => {
```

Comment:
I am thinking we won't be able to delete colors on the database per say (we can just make them inactive and not viewable to the users/admin). Otherwise we can't record what colors were used historically. Thoughts?

Reply:
Good point. I know you set up the 'isActive' boolean attribute so I'll add a 'not in use' section to the Figma designs